### DISCUSSION TITLE
Entity Voxels

### DISCUSSION THREAD

**Discussion by AddisonSchmidt** [Inspirateur/riverbed#45] [Ideas]
I think vehicles (such as trains) should be composed of smaller voxel grids independent of the world. The main difficulty posed by this is probably physics handling.

**Comment by Inspirateur** [comment]
Ok so this is a big one, I've thought about it and for now I don't plan on working on this. 

I'm not opposed to the idea, but it really requires a lot of work and i'd rather put this work on other big features for now 🤔

**Reply by Inspirateur** [reply]
(I've also moved this issue to the newly opened Discussion panel, which only has the "ideas" category for now, because this is not planned and requires more thinking)
